index.html

```diff
@@ -196,7 +196,7 @@ <h2>Extensibility</h2>
 </footer>
 
 <script type="module">
-import * as parsel from "./dist/parsel.js";
```

Comment:
Why? `./` should work fine? I'd rather avoid root-relative URLs if possible, as then it requires a localhost to run in the root of the repo to develop, rather than one at the root of all repos.

Reply:
Sorry, this was a leaky file.